ddtrace/internal/peer_service/processor.py

```diff
@@ -0,0 +1,34 @@
+from ddtrace.constants import SPAN_KIND
+from ddtrace.ext import SpanKind
+from ddtrace.internal.processor import SpanProcessor
+
+from . import PEER_SERVICE_ENABLED
+
+
+PEER_SERVICE_SOURCE_TAG = "_dd.peer.service.source"
+PEER_SERVICE_TAG = "peer.service"
+SPAN_KINDS_ENABLED_FOR_PEER_SERVICE = {SpanKind.CLIENT, SpanKind.PRODUCER}
+PRIORITIZED_PEER_SERVICE_DATA_SOURCES = ["messaging.kafka.bootstrap.servers", "db.name", "rpc.service", "out.host"]
```

Comment:
Should we make this configurable (ex: via an environment variable). Would customers want to add to this list?

Reply:
It's a good remark. Short answer is yes it will for sure come as a feature later but right now it has to be specified in order to have some behaviour across tracers

Reply:
for now its an internal-only API that's somewhat subject to change;  I'd recommend against making it configurable until its something we want to provide as an official option